src/classes/GE_FormRendererService.cls

```diff
@@ -234,9 +234,9 @@ public with sharing class GE_FormRendererService {
         }
 
         String relationshipField;
-        if (donorType == 'Account1') {
+        if (donorType == SObjectType.Account.Name) {
             relationshipField = SObjectType.Opportunity.fields.AccountId.Name;
-        } else if (donorType == 'Contact1') {
+        } else if (donorType == SObjectType.Contact.Name) {
```

Comment:
👍🏽

Reply:
We should be good here. Prior to this update, `donorType` was determined by the available values in the donor type picklist. This change moves the sourcing of `donorType` to a wired `getRecord` provided by the `uiRecordApi`. `getRecord` returns either an Account or Contact record based on the selected donor's id. The record object that's returned has an object api name property. That change is in lines 37 and 46 in `geReviewDonations.js`.

Additionally line 46 should be the one place where the aura enabled `getOpenDonations` method is called.